openassessment/xblock/openassessmentblock.py

```diff
@@ -119,9 +119,15 @@ class OpenAssessmentBlock(
     )
 
     allow_file_upload = Boolean(
-        default=False,
+        default=None,
+        scope=Scope.content,
+        help="Do not use. For backwards compatibility only."
```

Comment:
@muzaffaryousaf what do you mean? translate on help text "Do not use. For backwards compatibility only." Where should I do it?